cluster/saltbase/salt/docker/docker-defaults

```diff
@@ -1,3 +1,3 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false"
+DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false --log-level=warning"
```

Comment:
Here is the man page: 

https://github.com/docker/docker/blob/c45ad0b02d1cc1f88aaabab18aed27aeb20b9e7a/man/docker.1.md#L46